### PR TITLE
feat: make highlight textarea expandable

### LINKED ISSUE #1103: Bug: Use HTML form elements on the Highlights
### Describe the bug

Screen Shot 2023-04-18 at 7 26 37 AM

### Steps to reproduce

Form elements was suggested in slack. Make the box expand based on its contents.

### Affected services

insights.opensauced.pizza

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [ ] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by takanome-dev**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

This PR makes the highlight textarea expandable
## Related Tickets & Documents

Fixes #1103
## Mobile & Desktop Screenshots/Recordings

https://github.com/open-sauced/insights/assets/79809121/cd98555a-a6b2-462d-a50e-a8099a97ac2e

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by takanome-dev** [comment]
Oh forgot to test for the edit highlight, will fix it :sweat_smile:

**Review by bdougie — APPROVED** [review]
Thanks.

File: components/molecules/HighlightInput/highlight-input-form.tsx [hunk 1]
```diff
@@ -133,7 +133,7 @@ const HighlightInputForm = ({ refreshCallback }: HighlightInputFormProps): JSX.E
           </CollapsibleTrigger>
           <CollapsibleContent>
             <Textarea
-              className={`resize-none font-normal text-light-slate-11 mb-2 transition focus:outline-none rounded-lg ${
+              className={`resize-y min-h-[80px] font-normal text-light-slate-11 mb-2 transition focus:outline-none rounded-lg ${
```
[the remark this answers is not in the document]
Comment (takanome-dev) [reply components/molecules/HighlightInput/highlight-input-form.tsx]: I just inspect the textarea when it's open by default and the height is around `78` so I used `80px` :sweat_smile: What value do you think we should use for the min-height?

[the remark this answers is not in the document]
Comment (bdougie) [reply components/molecules/HighlightInput/highlight-input-form.tsx]: I am on an iPhone 10. I think it is fine to overlook. I think experience needs some design help overall.

**Review by OgDev-01 — APPROVED** [review]

**Pull request merged**